apis/core/v1alpha1/policies.go

```diff
@@ -30,3 +30,17 @@ const (
 	// will be retained when the managed resource is deleted.
 	ReclaimRetain ReclaimPolicy = "Retain"
 )
+
+// A DeletionPolicy determines what should happen to the underlying external
+// resource when a managed resource is deleted.
+type DeletionPolicy string
+
+const (
+	// DeletionOrphan means the external resource will orphaned when its managed
+	// resource is deleted.
+	DeletionOrphan DeletionPolicy = "Orphan"
```

Comment:
Why not reuse `Retain`?

Reply:
I felt 'Orphan' was more descriptive in this specific context. 'Retain it' could be interpreted as somehow retaining Crossplane control of the resource, while 'orphan it' seems clearer that the resource will be kept without Crossplane controlling it.

Reply:
Yeah, makes sense.